homeassistant/components/zwave_js/__init__.py

```diff
@@ -462,17 +465,17 @@ def async_on_node_removed(self, event: dict) -> None:
                 # Remove trailing comma if it's there
                 if identifier[-1] == ",":
                     identifier = identifier[:-1]
-                notification_msg = f"{notification_msg} {identifier}."
+                notification_msg = f"{notification_msg} {identifier}"
             else:
-                notification_msg = f"{notification_msg}."
+                notification_msg = f"{notification_msg}"
             async_create(
                 self.hass,
-                notification_msg,
+                f"{notification_msg}, reloading integration.",
```

Comment:
Do we need to reload the integration? Where does that happen?

If the device will be added again by the driver there will be an event and it should just work without reload, I'm thinking.

Reply:
to be honest I am not sure why I changed this notification. I don't think this change reloads the integration, it just removes the device. So I think I can just revert this and change it back